app/instance-initializers/configure-trackjs.js

```diff
@@ -1,34 +1,40 @@
 import Ember from 'ember';
 
 export function initialize(application) {
-  let trackJs = application.container.lookup('service:trackjs');
+  let trackJs    = application.container.lookup('service:trackjs');
   let appVersion = application.container.lookup('application:main').get('version');
 
   trackJs.configure({
     version: appVersion
   });
 
-  // http://docs.trackjs.com/Examples/Integrating_with_Ember
-  Ember.onerror = function (err) {
-    if (err.name === 'TransitionError') {
-      Ember.debug('Ignoring TransitionError:', err);
-    } else {
-      trackJs.track(err);
-      Ember.Logger.assert(false, err);
+  Ember.onerror = function (error) {
+    if (error.name === 'TransitionAborted') { return; }
+
+    if (error instanceof Error) {
+      trackJs.track(error);
+      Ember.Logger.error(error.stack);
+      return;
```

Comment:
:+1: If you could add some tests when all said and done that'd be great. I'd hate to see a regression on such gnarly behaviour.